values/camunda-local-bpm.yaml.gotmpl

```diff
@@ -3,7 +3,7 @@ app:
   group: backend
   container:
     image: hmctsprivate.azurecr.io/camunda/bpm
-    tag: latest
+    tag: pr-209-5a722c6-20210902180010
```

Comment:
is it deliberately hardcoded?

Reply:
yes, because it depends on the camunda-bpm pr, so until that one is not merged I can't change it to latest otherwise, it will break the local env.